lib/ci-helper.ts

```diff
@@ -1,5 +1,8 @@
 import * as fs from "fs";
+import * as path from "path";
 import * as util from "util";
+import type { PullRequestChangedFile, Repository
+    } from "@octokit/graphql-schema";
```

Comment:
Do we need to hide the graphql api?  It is not the same as the GitHub rest api where there is  more ORM type management.  Graphql requests are quite data specific so anything coming back from a `GitHubGlue` wrapper would look very much like what is returned now.  There are some tools to generate a type from the graphql query (similar to the hand-coded types in the `ci-helper.test.ts` test cases).  They use the query and the schema to create type files.  Would you want to integrate one of those tools into gitgitgadget?

Reply:
Another possibility is to move the new user check functions into a separate file/class.

Reply:
I'd rather keep this simple. To be honest, I would even have preferred to implement our own commit parser and read the commit history from the local repository (into which the commits would have to be fetched)...

But since you already have the GraphQL-based implementation, maybe just implement the minimal interface that we need? Then we can return an array of that interface type from a method in `GitHubGlue`. At least that's my current (admittedly hand-waving) thinking.